index.html

```diff
@@ -39,6 +39,190 @@ <h2 class="connect">
+                    <li class="items job-year">2021</li>
+                </ul>
+                <p class="canopy-p">
+                    A daily section of privately personalized reads: no accounts or signups required.
+                </p>
+                <div class="lang-div">
+                    <ul class="language">
+                        <li class="button-html">Html</li>
+                        <li class="button-css">CSS</li>
+                        <li class="button-js">Javascript</li>
+                      </ul>
+                </div>
+                
+                <div>
+                    <button class="projects-button" type="submit">See Projects</button>
+                </div>
+            </div>
+
+        </div>
+
+        <!-- Fourth Card -->
+
+        <div class="work-container">
+
+            <div class="project1">
+                <img src="images/multi-post-2.png" alt="Snapshoot Portfolio" class="snapimage">
+                <h1 class="tonic-text">Tonic</h1>
+                <ul class="exp-items">
+                    <li class="items canopy">CANOPY</li>
+                    <li class="items job-year"> Back End Dev</li>
+                    <li class="items job-year">2021</li>
+                </ul>
+                <p class="canopy-p">
+                    A daily section of privately personalized reads: no accounts or signups required.
+                </p>
+                <div class="lang-div">
+                    <ul class="language">
+                        <li class="button-html">Html</li>
+                        <li class="button-css">CSS</li>
+                        <li class="button-js">Javascript</li>
+                      </ul>
+                </div>
+                
+                <div>
+                    <button class="projects-button" type="submit">See Projects</button>
+                </div>
+            </div>
+
+
+        </div>
+
+        <!-- About Page -->
+
+        <section class="about-container">
+            <!-- <header>
+                <nav class="navbar">
+                        <a href="#" class="logo-link">My Logo</a>
+                    <div class="menu">
+                        <i class="fas fa-bars"></i>
+                    </div>
+                </nav>     
+            </header> -->
+            <section class="intro">
+                <h1 class="hero">About <br>
+                    Myself
+                </h1>
+                <p class="message">Thank you for stopping by. Please feel free to browse my portfolio. I'm a software developer and love creating interesting things. If you have an idea that you need help with. I'll be happy to help bring it to life! Reach out anytime!</p>
+                <h2 class="connect"> 
+                    <a href="#">Reach Out</a>
+                </h2>
+                <ul class="connent-icons">
+                    <li><a href="#"></a><img src="images/twitter.png" alt="twitter"></li>
```

Comment:
- [ ] Your webpage looks a bit different from the original design. Kindly make the necessary changes because respecting the original design is one of the project's requirements as it's an important skill for a front-end dev
Original design : 
![image](https://user-images.githubusercontent.com/43698511/133910757-dee8d9a4-52c7-4eaa-91e6-248c43b36999.png)
your webpage : 
![image](https://user-images.githubusercontent.com/43698511/133910763-4dd21fac-327b-4ff4-b5ae-c21b069f8e6a.png)
Kindly adjust the border-radius for the about me section to make it match the original design.
![image](https://user-images.githubusercontent.com/43698511/133910777-8085281c-1b56-4aab-b186-ee4e8ddf6ab9.png)
Kindly adjust the button and add the missing arrow icon for the cathegories